pkg/dashboard/healthcheck/server.go

```diff
@@ -45,22 +45,22 @@ func NewDashboardServer(logger logger.Logger,
 func (s *DashboardServer) Start() error {
 
 	// if we're disabled, simply log and do nothing
-	if !s.Enabled {
-		s.Logger.Debug("Disabled, not listening")
+	if !s.AbstractServer.Enabled {
+		s.AbstractServer.Logger.Debug("Disabled, not listening")
```

Comment:
Why is it not accessible directly anymore?
The AbstractServer is an anonymous property.

Reply:
@TomerShor it is accessible and code is compilable, however linter doesn't like it now because linting rules became way stricter.

Reply:
I think we can add an ignore rule to our golanci linter instead of changing all of this, since this is a major feature that we use extensively.

Reply:
@TomerShor I haven't found any ignore rules to ignore anonymous properties only in typecheck - it seems to be the same error if property really doesn't exist